2019-03-Autumn/2_string_manipulation/README.md

```diff
@@ -86,20 +107,159 @@ TODO :bug:
 
 TODO :bug:
 
-### 3. PROBLEM 3 TODO :bug:
-
-Source: TODO :bug:
-
-#### Naive/Simple Solution 
-
-TODO :bug:
-
-#### Optimal Solution
+### 3. Longest Substring
 
-TODO :bug:
-
-#### Testing The Solutions OR Driver For Solution
-
-TODO :bug:
+Source: [GeeksForGeeks](https://www.geeksforgeeks.org/longest-common-substring-dp-29/)
+
+#### Simple Solution 
+
+A naive approach to solving this problem takes the following steps:
+
+```
+1. Find all possible substring combinations of s1
+2. For each substring, check whether that substring exists in s2.
+3. Return identifiers for the longest substring that was found.
+```
 
+A code implementation in Python is as follows:
 
+```python3
+# a naive implementation
+def longest_substring(s1: str, s2: str) -> tuple:
+    indeces = (-1, -1)
```

Comment:
Just noticed your spelling for indices is wrong for the entire document lol

Reply:
TIL

Reply:
also, oof :sweat_smile: